index.js

```diff
@@ -2,24 +2,54 @@
 
 // Adapted from https://github.com/markdown-it/markdown-it/blob/fbc6b0fed563ba7c00557ab638fd19752f8e759d/docs/architecture.md
 
-function markdownitLinkAttributes (md, config) {
-  config = config || {}
+var optionKeys = ['ignoreRelative', 'pattern']
+var absoluteUrlPattern = /^([a-z]+:)?\/\//
```

Comment:
I assume you're doing it this way to support `data:`, `javascript:`, etc type links.

Reply:
I'll try